firestore-bigquery-export/firestore-bigquery-change-tracker/package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "@google-cloud/bigquery": "^4.7.0",
     "firebase-admin": "^7.1.1",
```

Comment:
I think the Admin SDK needs updating also here and is what's causing the failure